test.js

```diff
@@ -0,0 +1,9 @@
+import { Parcel } from "npm:@parcel/core";
+let bundler = new Parcel({
+  entries: "test.js",
+  defaultConfig: "@parcel/config-default",
+});
+console.log(bundler);
+let { bundleGraph, buildTime } = await bundler.run();
+let bundles = bundleGraph.getBundles();
+console.log(`✨ Built ${bundles.length} bundles in ${buildTime}ms!`);
```

Comment:
Can you turn it into a proper test?

Reply:
Hmm thats not a relevant test actually. Just leftover from another branch. Removed.